app.js

```diff
@@ -213,24 +192,19 @@
   function parsePurpleAirData(json) {
     let sensors = [];
     let fields = [];
-    fields["indoor"] = json.fields.findIndex((e) => e === "Type");
-    fields["latitude"] = json.fields.findIndex((e) => e === "Lat");
-    fields["longitude"] = json.fields.findIndex((e) => e === "Lon");
-    fields["id"] = json.fields.findIndex((e) => e === "ID");
-    fields["age"] = json.fields.findIndex((e) => e === "age");
+    fields["latitude"] = json.fields.findIndex((e) => e === "latitude");
+    fields["longitude"] = json.fields.findIndex((e) => e === "longitude");
+    fields["sensor_index"] = json.fields.findIndex((e) => e === "sensor_index");
     for (const sensor of json.data) {
-      // Ignore sensors which are either indoor or updated over 5 minutes ago
-      if (sensor[fields["indoor"]] === 0 && sensor[fields["age"]] < 5) {
```

Comment:
This is now done server side, via `location_type=0&max_age=300` appended to the full sensor list API requests 🎉